lib/ts/ink_defs.h

```diff
@@ -130,6 +145,12 @@ max(const T a, const T b)
 
 #if TS_USE_HWLOC
 #include <hwloc.h>
+#if HWLOC_API_VERSION < 0x00010b00
+#define HWLOC_OBJ_NUMANODE  HWLOC_OBJ_NODE
+#endif
+#elif __cplusplus
+// some signatures need this
+using hwloc_obj_type_t = int; 
```

Comment:
Is this how it's done in the rest of the code? Most of the places I've seen use the `typedef` form.